src/Http/Session.php

```diff
@@ -653,14 +653,13 @@ public function renew(): void
 
         $this->start();
         $params = session_get_cookie_params();
+        unset($params['lifetime']);
+        $params['expires'] = time() - 4200;
+        /** @phpstan-ignore-next-line */
         setcookie(
             (string)session_name(),
             '',
-            time() - 42000,
-            $params['path'],
-            $params['domain'],
-            $params['secure'],
-            $params['httponly'],
+            $params,
```

Comment:
@markusramsak Sorry, the annotation needs be be above this line, please remove the previous one.